packages/postgraphile-core/__tests__/pg10-data.sql

```diff
@@ -9,3 +9,10 @@ insert into pg10.network values
   (8, '::ffff:1.2.3.0/128', '::ffff:1.2.3.0/128', 'AABBCCDDEEFF', '08002b0102030405');
 
 alter sequence pg10.network_id_seq restart with 9;
+
+insert into pg10.types values (
+  12,
+  'postgraphile_test_authenticator', 
+  'b'
+);
+
```

Comment:
Resetting the sequence each time makes it so you can run the tests again without having to recreate the database schema.

```suggestion

alter sequence pg10.types_id_seq restart with 13;
```

Reply:
Ah, I forgot to set that to the correct number.

Reply:
This `types_id_seq` sequence is not reset, only `network_id_seq` is.

Reply:
You're absolutely right, and `network_type_id_seq` is set to the wrong number.